app/views/houses/members_letters.html.haml

```diff
@@ -6,8 +6,11 @@
 %h1= page_title
 
 = render partial: 'shared/id_a_to_z', locals: { route_symbol: :house_members_a_z_letter_path, id: @house.graph_id}
-%ul.list
-  = render partial: 'pugin/cards/person-list', locals: { people: @people }
+- if @people.empty?
```

Comment:
Reset this file back to how it was before - the front-end team have already implemented this inside of 'pugin/cards/person-list'